Dead From Beyond/deadfrombeyond.asl

```diff
@@ -46,12 +47,16 @@ start
 
 split
 {
-    return old.round != current.round &&
+    return (old.round != current.round &&
            settings.ContainsKey("round_" + current.round) &&
-           settings["round_" + current.round];
+           settings["round_" + current.round]) || 
+
+           settings.ContainsKey("ee") &&
```

Comment:
I have a few comments on this implementation, so bear with me - most of this has to do with convention or things that just sorta feel "right" to me, from experience. I truly apologise for the long paragraphs on what should just be a simple change, I have definitely felt frustration from people doing this to me in the past, but I want to explain a bit more in detail about each point. None of this comes from malice. Happy to chat about this more!

First, we should probably avoid complicating this return statement. I would do this like so, using another if statement:
```cs
split
{
    if ( ... ) // EE condition inside here
    {
        return true;
    }

    return ...; // original return statement
}
```
Even though logically it is the same, I like the conceptual separation here of having the "optional" or "not as important splits" above the "main" split. Really, this is just a way of trying to de-complicate the if statement.

Second, `settings.ContainsKey` actually checks if the given setting exists at all. Since we've defined this in `startup {}`, this always returns true. What we actually want is `settings["ee"]`. This is somewhat obscure, (the `ContainsKey` method isn't documented anywhere), but you can read the [AutoSplitter documentation](https://github.com/LiveSplit/LiveSplit.AutoSplitters) if you haven't already.

Third, the mechanism for checking the scene change _works_, but isn't very robust nor readable. I think the logic you have here is equivalent to: if we are in a level scene, and we just transitioned from a scene that is both not a level scene and not the main menu, then split.

That works because "a scene that is both not a level scene and not the main menu" only applies to one scene, the easter egg scene. This is not obvious, and is susceptible to updates adding unexpected scenes! Also, there's no need to check we're transitioning to _any_ level - we only want Level 2. So, we should just say:
```cs
old.activeScene == "the name of the easter egg scene" &&
current.activeScene == "whatever level 2's scene name is";
```

This is much more readable, and it's more obvious what is actually going on.